Documentation/ColumnsConfig/Type/Inline/Properties/ForeignSelector.rst

```diff
@@ -15,4 +15,5 @@ foreign\_selector
    It will be rendered as a multi-select-box. On clicking on an item inside the selector a new relation is created.
    The :code:`foreign_selector` points to a field of the :ref:`foreign_table <columns-inline-properties-foreign-table>`
    that is responsible for providing a selector-box – this field on the :code:`foreign_table` usually
-   is of type :ref:`select <columns-select>` and also has a :code:`foreign_table` defined.
+   is of type :ref:`select <columns-select>` and also has a :code:`foreign_table` defined. If the foreign table
```

Comment:
You would  not use the MM table here directly but link to a foreign table using a MM relation stored in an intermediate table. Please rephrase